tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -387,8 +387,11 @@ public function testInvalidAnnotationUsageButIgnoredClass()
         $reader = $this->getReader();
         $ref = new \ReflectionClass('Doctrine\Tests\Common\Annotations\Fixtures\InvalidAnnotationUsageButIgnoredClass');
         $annots = $reader->getClassAnnotations($ref);
-
-        $this->assertEquals(2, count($annots));
+        if ($annots[0] instanceof IgnoreAnnotation) {
```

Comment:
I have removed if statement from the unittest and moved testing the behaviour of SimpleAnnotationReader into the SimpleAnnotationReaderTest